src/Summoner/CLI.hs

```diff
@@ -247,11 +247,24 @@ $endLine
 artFooter :: Text
 artFooter = [text|
 $endLine
-          ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
-  ________┃                                  ┃_______
-  ╲       ┃   λ Make Haskell Great Again λ   ┃      ╱
-   ╲      ┃                                  ┃     ╱
-   ╱      ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛     ╲
-  ╱__________)                            (_________╲
+              , *   +
+           +      o   *             ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+            * @ ╭─╮  .      ________┃                                 ┃_______
+           ╱| . │λ│ @   '   ╲       ┃   λ Haskell's summon scroll λ   ┃      ╱
+         _╱ ╰─  ╰╥╯    O     ╲      ┃                                 ┃     ╱
+        .─╲"╱. * ║  +        ╱      ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛     ╲
+       ╱  ( ) ╲_ ║          ╱__________)                           (_________╲
+       ╲ ╲(')╲__(╱
+       ╱╱`)╱ `╮  ║
+ `╲.  ╱╱  (   │  ║
+  ╲.╲╱        │  ║
+  `╰══════════╯
+
+
+
+
+
+
+
```

Comment:
Maybe these empty lines are redundant and can be replaced with single `$endLine`?